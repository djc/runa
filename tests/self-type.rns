class test:
	def __init__(i: int):
		return
