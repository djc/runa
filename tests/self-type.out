tests/self-type.rns [2.24]: first method argument must be of type 'test'
    def __init__(self: int):
                       ^
